TYPO3.Neos/Resources/Private/Styles/Shared/_Select2.scss

```diff
@@ -198,7 +197,7 @@
 
 			i {
 				position: absolute;
-				left: 14px;
+				left: 13px;
```

Comment:
in my Chrome browser on Windows `left: 15px` seems to dead center the `x` inside the button